modules/calcite/src/test/sql/types/blob/test_blob.test_ignore

```diff
@@ -1,96 +1,106 @@
 # name: test/sql/types/blob/test_blob.test
 # description: BLOB tests
 # group: [blob]
-# Ignore https://issues.apache.org/jira/browse/IGNITE-15618
+# Ignore https://issues.apache.org/jira/browse/IGNITE-15123
 
 statement ok
 PRAGMA enable_verification
 
 statement ok
-CREATE TABLE blobs (b binary);
+CREATE TABLE blobs (b varbinary);
 
 # Insert valid hex strings
 statement ok
-INSERT INTO blobs VALUES('\xaa\xff\xaa'), ('\xAA\xFF\xAA\xAA\xFF\xAA'), ('\xAA\xFF\xAA\xAA\xFF\xAA\xAA\xFF\xAA')
+INSERT INTO blobs VALUES(x'aaffaa'), (x'AAFFAAAAFFAA'), (x'AAFFAAAAFFAAAAFFAA')
 
 query T
-SELECT * FROM blobs
+SELECT * FROM blobs ORDER BY b
```

Comment:
`ScriptRunner` supports an unordered comparison. You could simply define a query like `query T rowsort`. In that case all result will be sorted on a client side in a lexicographical order of an ASCII representation of the row. You could read more about it in a javadoc of `ScriptTestSuite`.

I believe, it would be better to use unordered comparison than modify an original query.